### DISCUSSION TITLE
Add `--files` option; bugfixes; use Go 1.19

### DISCUSSION THREAD

**Discussion by johnkerl** [johnkerl/miller#1444] [Announcements]
## Features

* Add a `--files` option by @johnkerl in https://github.com/johnkerl/miller/pull/1426

## Bugfixes

* Fix ragged-CSV auto-pad by @johnkerl in https://github.com/johnkerl/miller/pull/1428
* Absent variable on left side of boolean OR (`||`) expression makes it absent by @johnkerl in https://github.com/johnkerl/miller/pull/1434
* Include `null` in `any` typemask by @johnkerl in https://github.com/johnkerl/miller/pull/1395
* transformers/grep: avoid allocations with `(*regexp.Regexp).MatchString` by @Juneezee in https://github.com/johnkerl/miller/pull/1416
* JSONL output does not properly handle keys with quotes by @johnkerl in https://github.com/johnkerl/miller/pull/1425

## Minor changes

* Update to Go 1.19 by @johnkerl in https://github.com/johnkerl/miller/pull/1441
* miller 6.10.0 by @johnkerl in https://github.com/johnkerl/miller/pull/1442
* Add `winget` to `README.md` by @rursprung in https://github.com/johnkerl/miller/pull/1414
* Name-neaten for #1392 by @johnkerl in https://github.com/johnkerl/miller/pull/1393

## Miller as API

* Export library code in `pkg/` by @johnkerl in https://github.com/johnkerl/miller/pull/1391
* Better API example by @johnkerl in https://github.com/johnkerl/miller/pull/1392

## Dependencies

* Bump golang.org/x/text from 0.12.0 to 0.13.0 by @dependabot in https://github.com/johnkerl/miller/pull/1382
* Bump golang.org/x/sys from 0.11.0 to 0.12.0 by @dependabot in https://github.com/johnkerl/miller/pull/1381
* Bump golang.org/x/term from 0.11.0 to 0.12.0 by @dependabot in https://github.com/johnkerl/miller/pull/1380
* Bump actions/checkout from 3.6.0 to 4.0.0 by @dependabot in https://github.com/johnkerl/miller/pull/1383
* Bump goreleaser/goreleaser-action from 4.4.0 to 4.6.0 by @dependabot in https://github.com/johnkerl/miller/pull/1385
* Bump actions/upload-artifact from 3.1.2 to 3.1.3 by @dependabot in https://github.com/johnkerl/miller/pull/1387
* Bump actions/cache from 3.3.1 to 3.3.2 by @dependabot in https://github.com/johnkerl/miller/pull/1390
* Bump goreleaser/goreleaser-action from 4.6.0 to 5.0.0 by @dependabot in https://github.com/johnkerl/miller/pull/1396
* Bump actions/checkout from 4.0.0 to 4.1.0 by @dependabot in https://github.com/johnkerl/miller/pull/1400
* Bump golang.org/x/term from 0.12.0 to 0.13.0 by @dependabot in https://github.com/johnkerl/miller/pull/1404
* Bump github.com/mattn/go-isatty from 0.0.19 to 0.0.20 by @dependabot in https://github.com/johnkerl/miller/pull/1411
* Bump actions/checkout from 4.1.0 to 4.1.1 by @dependabot in https://github.com/johnkerl/miller/pull/1412
* Bump golang.org/x/text from 0.13.0 to 0.14.0 by @dependabot in https://github.com/johnkerl/miller/pull/1419
* Bump golang.org/x/sys from 0.13.0 to 0.14.0 by @dependabot in https://github.com/johnkerl/miller/pull/1420
* Bump golang.org/x/term from 0.13.0 to 0.14.0 by @dependabot in https://github.com/johnkerl/miller/pull/1423
* Bump golang.org/x/term from 0.14.0 to 0.15.0 by @dependabot in https://github.com/johnkerl/miller/pull/1432
* Bump actions/setup-go from 4.1.0 to 5.0.0 by @dependabot in https://github.com/johnkerl/miller/pull/1436

## New Contributors
* @rursprung made their first contribution in https://github.com/johnkerl/miller/pull/1414

**Full Changelog**: https://github.com/johnkerl/miller/compare/v6.9.0...v6.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/johnkerl/miller/releases/tag/v6.10.0'>Add --files option; bugfixes; use Go 1.19</a>.</em>

**Comment by aborruso** [comment]
Thank you very much
